code/_onclick/cyborg.dm

```diff
@@ -83,6 +83,14 @@
 		if(CanReach(A,W))
 			W.melee_attack_chain(src, A, params)
 			return
+		else
+			if(LAZYACCESS(modifiers, RIGHT_CLICK))
+				var/after_attack_secondary_result = W.afterattack_secondary(A, src, FALSE, params)
+
+				if(after_attack_secondary_result == SECONDARY_ATTACK_CALL_NORMAL)
+					W.afterattack(A, src, FALSE, params)
+			else
+				W.afterattack(A, src, FALSE, params)
```

Comment:
```suggestion
// BUBBER EDIT - START
		else
			if(LAZYACCESS(modifiers, RIGHT_CLICK))
				var/after_attack_secondary_result = W.afterattack_secondary(A, src, FALSE, params)

				if(after_attack_secondary_result == SECONDARY_ATTACK_CALL_NORMAL)
					W.afterattack(A, src, FALSE, params)
			else
				W.afterattack(A, src, FALSE, params)
// BUBBER EDIT - END
```
May need this. I'm not entirely sure if the crusher itself is a skyrat or a TG thing. I'm leaning skyrat and I'm unsure if there's a need on TG itself to justify this being an upstream.

Reply:
It's a TG issue actually and this is marked as DNM to see if it causes any issues before I upstream it

Reply:
> Reference

👍

Reply:
Cyborg crushers are a tg thing but the crusher itself is not